webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/TextValue/TextValuePresentation.tsx

```diff
@@ -146,41 +149,71 @@ export const TextValuePresentation: TabContainerPanelComponent<IDataValuePanelPr
       }
     }
 
-    if (!activeTabs.some(tab => tab.key === state.currentContentType)) {
-      const contentType = activeTabs.length > 0 && activeTabs[0].key ? activeTabs[0].key : TEXT_PLAIN_TYPE;
-      state.setContentType(contentType);
+    async function selectTabHandler(tabId: string) {
+      // currentContentType may be selected automatically we don't want to change state in this case
+      if (tabId !== contentType) {
+        state.setContentType(tabId);
+      }
+    }
+
+    function toggleLineWrappingHandler() {
+      state.setLineWrapping(!lineWrapping);
     }
 
     return styled(style)(
-      <container>
-        <actions>
-          <TabsState
-            dataFormat={dataFormat}
-            resultIndex={resultIndex}
-            container={textValuePresentationService.tabs}
-            currentTabId={state.currentContentType}
-            model={model}
-            lazy
-            onChange={tab => state.handleTabOpen(tab.tabId)}
-          >
-            <TabList style={[BASE_TAB_STYLES, styles, UNDERLINE_TAB_STYLES]} />
-          </TabsState>
-        </actions>
-        <EditorLoader key={isReadonly ? '1' : '0'} value={textValue} readonly={isReadonly} extensions={extensions} onChange={handleChange} />
-        {isTruncated && <QuotaPlaceholder limit={limit} size={valueSize} />}
-        <div className={s(moduleStyle, { toolsContainer: true })}>
+      <Container vertical gap dense overflow>
+        <Container keepSize center overflow>
+          <Container keepSize>
```

Comment:
in this case it's because i don't want to add extra styles to `TabsList`
I suppose it's not always about problem in the `Container` but in matching other components together